sparkbox/bouncy-ball
# chore: update snowpack 2.x to esinstall

If we try to update snowpack 2.x → 3.x, it'll change the bundling approach dramatically. Snowpack 3.x is more like webpack which runs a dev server, and file watcher, building files on each file change.

Instead, we can update snowpack 2.x to esinstall, which preserves the 'build once at install-time' approach that we had before.

### Testing instructions:
1. Pull down this branch
2. Run `npm install`
3. Run `npm build` and confirm that the build command works as expected (the files in `web_modules` should be rebuilt, but there should be no changes, since I've already included those changes on this branch).
4. Run `npm start` and click around the page to confirm that everything still works as expected.
